pkg/util/util.go

```diff
@@ -1,7 +1,13 @@
 package util
 
 import (
+	"fmt"
+	"strings"
+
 	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/log"
+	clientutil "kubevirt.io/client-go/util"
+	"kubevirt.io/kubevirt/pkg/virt-operator/creation/rbac"
```

Comment:
Could you leave the rbac related helper function in the webhooks packages? This here creates a binding across the whole project to the operator otherwise. Can make changes difficult.

Reply:
@rmohr Done